katharsis-jpa/src/main/java/io/katharsis/jpa/query/querydsl/QuerydslExpressionFactory.java

```diff
@@ -0,0 +1,9 @@
+package io.katharsis.jpa.query.querydsl;
+
+import com.querydsl.core.types.Expression;
+import com.querydsl.jpa.impl.JPAQuery;
+
+public interface QuerydslExpressionFactory<T extends Expression<?>> {
+
+	public Expression<?> getExpression(T parent, JPAQuery<?> jpaQuery);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1452)